ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-isis.j2

```diff
@@ -8,6 +8,9 @@ router isis {{ router_isis.instance }}
 {%     if router_isis.is_type is arista.avd.defined %}
    is-type {{ router_isis.is_type }}
 {%     endif %}
+{%     if router_isis.advertise_passive_only is arista.avd.defined(true) %}
```

Comment:
this is already covered in line 32-34 with a slightly different variable structure

Reply:
Removed this.